lib/Models/Cesium.ts

```diff
@@ -726,7 +728,8 @@ export default class Cesium extends GlobeOrMap {
           }
 
           if (target.mapItems.length > 0) {
-            // Zoom to the first item!
+            // Zoom to the first item after setting the timeline to use the target.
+            if (TimeVarying.is(target)) that.setAsTimelineClockSource(target);
             return that.zoomTo(target.mapItems[0], flightDurationSeconds);
```

Comment:
We want to use the target item as the timeline source if it is TimeVarying. Mimicks the behavior in v7.